tests/unit_tests/storage/adapter/test_nanobind_adapter.cpp

```diff
@@ -8,22 +8,31 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+#include "nanobind/nanobind.h"
+#include <Python.h>
 #include <array>
 
 #include <gtest/gtest.h>
```

Comment:
```suggestion
#include <array>

#include <gtest/gtest.h>
#include <Python.h>
```

Nanobind is hopefully already included in `nanobind_adapter.hpp`. AFAIR, at some point we used to have the tested file (i.e., `nonbind_adapter.hpp`) at the top of the includes, to make sure that the tested file includes all required dependencies correctly.